docs/services/search/_index.md

```diff
@@ -7,4 +7,10 @@ geekdocFilePath: _index.md
 geekdocCollapseSection: true
 ---
 
+## Abstract
+
 This service provides search functionality.
```

Comment:
It is obvious that a search service provides search functionality, but could we have a bit more meat on the bone 😅

Reply:
Please let us do abstracts in subsequent PRs. They should be more than one sentence.